databricks/koalas/indexes.py

```diff
@@ -882,10 +882,21 @@ def dropna(self):
     def unique(self, level=None):
         """
         Return unique values in the index.
+
         Be aware the order of unique values might be different than pandas.Index.unique
 
-        :param level: int or str, optional, default is None
-        :return: Index without deuplicates
+        Parameters
+        ----------
+        level : int or str, optional, default is None
+
+        Returns
+        -------
+        Index without deuplicates
```

Comment:
Maybe `deuplicates ` -> `duplicates` ?

Reply:
oops, sorry - I missed this.

Reply:
I'm gonna include this fixing to https://github.com/databricks/koalas/pull/1412 my yet unmerged PR.

Reply:
oops, thanks!